examples/terminating-gateway-tls/README.md

```diff
@@ -0,0 +1,141 @@
+# Terminating Gateways on ECS.
+
+This example demonstrates accessing non mesh destinations services from mesh tasks with the help of terminating gateways deployed as ECS tasks.
```

Comment:
You could add information about how the terminating gateway performs mTLS with the external server app.